st2reactor/st2reactor/container/sensor_wrapper.py

```diff
@@ -264,8 +265,10 @@ def _get_sensor_instance(self):
                                                         file_path=self._file_path,
                                                         class_name=self._class_name)
         except Exception as e:
-            msg = ('Failed to load sensor class from file "%s"'
-                   ' (sensor file most likely doesn\'t exist): %s' % (self._file_path, str(e)))
+            tb_msg = traceback.format_exc()
+            msg = ('Failed to load sensor class from file "%s" (sensor file most likely doesn\'t '
+                   'exist or contains invalid syntax): %s' % (self._file_path, str(e)))
+            msg += '\n\n' + tb_msg
             raise ValueError(msg)
```

Comment:
While we are on this, do you want to raise a specific exception than ValueError?